lib/Transforms/OptimizeRelinearization/OptimizeRelinearization.td

```diff
@@ -38,6 +38,13 @@ def OptimizeRelinearization : Pass <"optimize-relinearization"> {
            /*default=*/"false",
            "When true, the ILP uses op source locations in variable names, "
            "which can help debug ILP model bugs.">,
+    Option<"allowMixedLevelOps",
+           "allow-mixed-level-ops",
+           "bool",
+           /*default=*/"true",
+           "When true, allow ops to have mixed-level ciphertexts as inputs, e.g., "
+           "adding two ciphertexts with different key bases; this is supported by "
+           "many FHE backends">,
```

Comment:
Since we extensively use the term "degree" in ILP model and doc, the name "allow-mixed-degree-ops" could be more consistent, as pointed out by Alex in https://github.com/google/heir/issues/1284#issuecomment-2609796156

mixed-level/cross-level ct op often refers to different RNS level in the literature.

Reply:
Done